### PR TITLE
Added option to use ip-freebind.

### PR DESCRIPTION
**Pull request by tmichelis**
Hej,

we have to use ip-freebind instead of interface-automatic as it is still marked as experimental. We would like to use this module from puppet-forge instead of maintaining it by ourselves. That's why I ask for this pull request.

Cheers
Tobias

### CODE REVIEW

**Comment by xaque208** [comment]
HI @tmichelis, Thanks for the PR.  Can you check the latest code to see if this is still required?  This will need a rebase at least due to conflicts that were just merged in.

**Comment by tmichelis** [comment]
Yes I will. I will come back to you. Thanks you

**Comment by b4ldr** [comment]
This was added in https://github.com/xaque208/puppet-unbound/pull/156 so you should be good now

File: templates/unbound.conf.erb [hunk 1]
```diff
@@ -21,7 +21,7 @@ server:
 <% end -%>
   log-time-ascii: <%= bool2yesno(@log_time_ascii) %>
   log-queries: <%= bool2yesno(@log_queries) %>
-  log-replies: <%= bool2yesno(@log_replies) %>
+  #log-replies: <%= bool2yesno(@log_replies) %>
```
Comment (b4ldr) [inline templates/unbound.conf.erb]: why comment this out, did you see an issue?

**Comment by xaque208** [comment]
@tmichelis Hi.  Are you able to come back to this work?

**Pull request closed without merging**